.github/workflows/ci.yml

```diff
@@ -24,6 +20,13 @@ jobs:
       matrix:
         go-version: [1.12.x, 1.13.x]
     steps:
+      - name: Install CouchDB
+        run: |
+          sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 379CE192D401AB61
+          echo "deb https://apache.bintray.com/couchdb-deb bionic main" | sudo tee -a /etc/apt/sources.list
+          sudo apt-get update
+          echo -ne "couchdb couchdb/mode select standalone\ncouchdb couchdb/mode seen true\ncouchdb couchdb/bindaddress string 127.0.0.1\ncouchdb couchdb/bindaddress seen true\ncouchdb couchdb/adminpass password \ncouchdb couchdb/adminpass seen true\ncouchdb couchdb/adminpass_again password \ncouchdb couchdb/adminpass_again seen true\n" | sudo debconf-set-selections
+          DEBIAN_FRONTEND=noninteractive sudo apt-get install -y --force-yes couchdb
```

Comment:
NIT: would be easilly readable with a here-document:

```
sudo debconf-set-selections <<-EOF
    couchdb couchdb/mode select standalone
    couchdb couchdb/mode seen true
    couchdb couchdb/bindaddress string 127.0.0.1
    couchdb couchdb/bindaddress seen true
    couchdb couchdb/adminpass password
    couchdb couchdb/adminpass seen true
    couchdb couchdb/adminpass_again password
    couchdb couchdb/adminpass_again seen true
EOF
```

note: leading spaces are tabs. tabs are stripped at the beginning of lines with `<<-` here-documents operator

Reply:
unless gh actions interpret each line as a separate command :(

Reply:
Let's try. I have made several attempts to make it works, and I wanted to avoid this potential issue while debugging, but now that I have a version that works, we cant try to improve it.

Reply:
It works!

Reply:
Nice !